### PR TITLE
Migrate from fasit to vault

### PR DESCRIPTION
**Pull request by ybelMekk**
This change is pretty major for the whole project. 

Think this one need to be merged in before the other pull request "Admin groups".

I also updated dep. versions in build.gradle.kts. Bumped Kotlin ver. 1.3.21 among others.

Also bumped gradle wrapper properties to latest, 5.3.

Set up for vault is done, for test, pre and prod. @tronghn , probably you have to dobbel check this ones with properties from fasit also if default env. is right set. Settings should be ok.

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+}
+
+val generatedSourcesDir = "$buildDir/generated-sources"
+
+tasks {
+    create("printVersion") {
+        println(project.version)
+    }
+    withType<ShadowJar> {
+        classifier = ""
```
Comment (MikAoJk) [inline build.gradle.kts]: Maybe remove this, its Deprecated

File: build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        transform(ServiceFileTransformer::class.java) {
+            setPath("META-INF/cxf")
+            include("bus-extensions.txt")
+        }
+    }
+    withType<Test> {
+        useJUnitPlatform {
+            includeEngines("spek2")
+        }
+        testLogging.events("passed", "skipped", "failed")
```
Comment (MikAoJk) [inline build.gradle.kts]: Use:
```
testLogging {
            showStandardStreams = true
        }
```
instead.
Tells whether output on standard out and standard error will be logged. Equivalent to checking if both log events TestLogEvent.STANDARD_OUT and TestLogEvent.STANDARD_ERROR are set.
ref: https://docs.gradle.org/current/dsl/org.gradle.api.tasks.testing.logging.TestLoggingContainer.html#org.gradle.api.tasks.testing.logging.TestLoggingContainer:showStandardStreams

File: build.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+val unboundidVersion = "4.0.10"
+
+val swaggerVersion = "3.22.0"
+
+val spekVersion = "2.0.2"
+val kluentVersion = "1.49"
+
+val konfigVersion = "1.6.10.0"
+
+val mainClass = "no.nav.integrasjon.MainKt"
```
Comment (MikAoJk) [inline build.gradle.kts]: Maybe move to mainClass to  tasks, like this:
 ```
 withType<Jar> {
        manifest.attributes["Main-Class"] = "no.nav.integrasjon.MainKt"
    }
```

File: gradle/wrapper/gradle-wrapper.properties [hunk 4]
```diff
@@ -1,6 +1,6 @@
-#Wed Nov 28 20:24:17 CET 2018
+#Tue Apr 16 15:06:26 CEST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.3-all.zip
```
Comment (MikAoJk) [inline gradle/wrapper/gradle-wrapper.properties]: Bump gradle version to 5.4?

File: build.gradle.kts [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+        }
+    }
+    withType<Test> {
+        useJUnitPlatform {
+            includeEngines("spek2")
+        }
+        testLogging.showStandardStreams = true
+    }
+    withType<Wrapper> {
+        gradleVersion = "5.3"
```
Comment (MikAoJk) [inline build.gradle.kts]: Bump gradle version to 5.4?

**Review by MikAoJk — APPROVED** [review]

**Comment by tronghn** [comment]
Superseded by #30

**Pull request closed without merging**